.github/CODEOWNERS

```diff
@@ -1,4 +1,11 @@
-* @thewca/software-team
+# Actual contents that directly reflect on the documents published on the WCA website
 documents/ @thewca/board
 edudoc/ @thewca/wqac
 editable-image-files/ @thewca/wqac
```

Comment:
Please also add this for the WQAC: `documents-guide.md @thewca/wqac`

Reply:
Done :)

Reply:
Thanks!